Dockerfile.evm

```diff
@@ -1,29 +1,32 @@
 FROM docker.io/paritytech/ci-linux:production as builder
 LABEL description="This is the build stage for Parallel. Here we create the binary."
 
-ARG PROFILE=production
+ARG PROFILE=release
 ARG BIN=parallel
 
 WORKDIR /parallel
 
 COPY . /parallel
 
+RUN rustup default nightly
```

Comment:
I feel this will still break the docker image build in the future, rust nightly is very unstable. Are we able to use `rust-toolchain` or use `ci-linux:<commit>`

Reply:
nightly here means toolchain from docker.io/paritytech/ci-linux:production and now is 2022-07-25 https://github.com/paritytech/scripts/blob/1c0fde6a6eb6b1a700da7261b3b6be0463d26d17/dockerfiles/ci-linux/Dockerfile#L24

this should be tested by parity team and I think always sync with them should be fine. If anything breaking in our code not compatible with it we can just make some fix